OpenSim/Simulation/Manager/Manager.cpp

```diff
@@ -733,26 +695,16 @@ integrate(SimTK::State& s, double finalTime)
         if (checkHalt()) break;
     }
     finalize(_integ->updAdvancedState());
```

Comment:
It would be good to create a separate issue about determining if this should use `_integ->getState()` (if such an issue does not exist yet). No need to address in this PR.

Reply:
Opened up an issue for this.